docs/releases/5.0.txt

```diff
@@ -529,7 +529,34 @@ Should become::
         uuid = Char32UUIDField(primary_key=True, default=uuid.uuid4)
 
 Running the :djadmin:`makemigrations` command will generate a migration
-containing a no-op ``AlterField`` operation.
+containing a no-op ``AlterField`` operation. Since any new ``UUIDField``
+would use a ``UUID`` column, the ``Char32UUIDField`` should be used
+instead, when mixing column types is not desired.
+
+----
+
+Alternatively, existing ``UUIDField`` columns could be altered in the
+database to use a ``UUID`` column, by running a ``RunSQL`` operation in
+a data migration::
+
+  class Migration(migrations.Migration):
+      ...
+      operations = [
+          migrations.RunSQL(
```

Comment:
Should this be a no-op for the backwards migration and have `elidable=True` so it disappears when squashing?